samples/matter/common/src/bt_nus_service.cpp

```diff
@@ -146,8 +146,30 @@ bool NUSService::RegisterCommand(const char *const name, size_t length, CommandC
 
 void NUSService::DispatchCommand(const char *const data, uint16_t len)
 {
+	static constexpr char LF = '\n';
+	static constexpr char CR = '\r';
+	static const char *CRLF = "\r\n";
+
+	const size_t rawLenEol = len - 1;
+	const size_t rawLenWindowsEol = len - 2;
+
+	const char eol = *(data + rawLenEol);
```

Comment:
Is it guaranteed that `len > 0`? If not, it's safer to dereference after the size-check.